openstef_dbc/services/weather.py

```diff
@@ -55,6 +55,7 @@ def _get_nearest_weather_location(
         location: Union[Tuple[float, float], str],
         country: str = "NL",
         threshold: float = 150.0,
+        k: int = 1,
```

Comment:
Could you make this parameter name more descriptive? E.g. number_locations?

Reply:
Hi @clara-de-smet,
the parameter k has been renamed to number_locations :
https://github.com/OpenSTEF/openstef-dbc/blob/009e60fed4524dfa1c29e58d1a4d994422a931c7/openstef_dbc/services/weather.py#L53-L59